great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -217,10 +218,24 @@ def _build_parameters(
             variables=variables,
             parameters=parameters,
         )
-        if not (0.0 <= false_positive_rate <= 1.0):
+
+        if false_positive_rate < 0 or false_positive_rate > 1:
```

Comment:
@NathanFarmer Just noticed -- I liked the previous statement, using the floating point numbers, better -- would you mind reverting it?  Your error message is OK.  Thank you.

Reply:
✅